utils/_context/_scenarios/__init__.py

```diff
@@ -530,7 +530,12 @@ def all_endtoend_scenarios(test_object):
     )
     tracing_config_nondefault_3 = EndToEndScenario(
         "TRACING_CONFIG_NONDEFAULT_3",
-        weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false"},
+        weblog_env={
+            "DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false",
+            "DD_TRACE_CLIENT_IP_HEADER": "custom-ip-header",
+            # disable ASM to test non asm client ip tagging
```

Comment:
There is a `appsec_enabled` argument to `EndToEndScenario`, you can set it to `False`